sr_port/ydberrors_ansi.h

```diff
@@ -18,4 +18,18 @@ const static readonly int error_ansi[] = {
 	   0,	/* LIBYOTTAMISMTCH */
```

Comment:
This module is unnecessary. But it keeps coming back because of the way the git commit hook is currently written. Since it does not affect anything, I decided to stop manually deleting it every time it gets regenerated.